### ISSUE TITLE
Should there be boundary sounds for spin buttons, similar to mass location sliders?

### ISSUE THREAD

**Issue by emily-phet** [phetsims/gravity-force-lab-basics#152]
@terracoda brought up (from consultant discussion) that the spin buttons don't have a boundary sound, but the mass location sliders do. 

@Ashton-Morris suggested a solution (having a gentle 'error' sound if spin  buttons are pressed while inactive).

I think there is a deeper issue at play here - where visually the grab/drag of the mass objects and the press of the number spinner feel very different, but are actually very similar from the perspective of alternative input. This issue will come up across other sims...so I would prefer to hold off on digging deeply on this issue for this sim, and instead tackle it the next time we encounter it.

**Comment by terracoda** [comment]
@emily-phet, I agree this is a deeper issue, and keep in mind for Gravity Force Lab regular, the position sliders and the mass sliders are exactly the same interaction (native HTML input type range). 

We can explore this issue more deeply in that sim. 

I would like to mention that for the position sliders, I also designed an alert that fires on the first try to go beyond an edge (beyond edge of track, or closer than closest to the other sphere), but I did not do a similar alert for the the mass controls. I've been thinking that might also be worth investigating. I'll open an issue for that in GFL regular.

**Comment by terracoda** [comment]
Marking on hold so it is clear we will explore this idea further in GFL Regular.

**Comment by zepumph** [comment]
Would it be helpful to transfer this issue to gravity-force-lab? Github makes that easy if we want to do that.

**Comment by terracoda** [comment]
I think that would be fine.

**Comment by terracoda** [comment]
@emily-phet, opened the issue. @emily-phet  are you ok with moving the other repo?
